aws/outputs.tf

```diff
@@ -6,6 +6,10 @@ output "docker.private.ip" {
   value = "${aws_instance.tsuru-docker.private_ip}"
 }
 
+output "docker-registry.private_ip" {
```

Comment:
The commit f318c0f231086dc7e122ff504607c71dc9fca7b9 that it belongs to explains why.